api_tests/tests/orderbook.ts

```diff
@@ -7,10 +7,21 @@ import OrderFactory from "../src/actors/order_factory";
 import { twoActorTest } from "../src/actor_test";
 import { Entity, Link } from "comit-sdk/dist/src/cnd/siren";
 import { sleep } from "../src/utils";
+import SwapFactory from "../src/actors/swap_factory";
 
 it(
-    "orderbook_bob_makes_order_alice_takes_order",
+    "orderbook_bob_makes_order_alice_takes_order_both_redeem",
```

Comment:
This test is pretty long and verbose. It is quite hard to understand what is being tested here.

Ideally, the test reads exactly according to the mental model of what you are testing. For orderbook + doing a swap, that is:

- Bob makes an order
- Alice looks at her orders and takes one
- Alice funds
- Bob funds
- Alice redeems
- Bob redeems

It is hard to avoid all boilerplate due to the need for setting up stuff etc but we should at least aim as high as we can!
Conceptually, this test is not a whole lot different to the others. Just instead of saying `actor.create` we use the orderbook to `make` and `take` orders.
Hence, our target should be to make this test roughly as concise as the other e2e tests.

Let me know if you need help with that :)